packages/react-provider/src/index.tsx

```diff
@@ -19,7 +19,7 @@ export const registerReducer = _registerReducer;
 export type ICourierMessage = TransportTypes.ICourierMessage;
 export type ICourierContext = types.ICourierContext;
 export const CourierContext = React.createContext<ICourierContext | undefined>(
```

Comment:
could you tell me how setting context to `undefined` different than not setting anything?

Reply:
typescript got mad if you didn't.  basically you have 2 choices.  make the context with a context object or make it with undefined.  There isn't a way to say ICourierContext is optional `React.createContext<ICourierContext?>` so you have to `| undefined`

Reply:
Makes sense. Resolving